changelogs/unreleased/gh-9556-fix-uri-ipv6.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/uri
```

Comment:
Nit: The `uri: fix parsing IPv6` feels a bit too general. I would make the title more certain. For example, `uri: fix parsing uppercased IPv6 addresses`.

Reply:
commit message changed to suggested